provisioning/miniprov/hare_mp/main.py

```diff
@@ -83,31 +83,49 @@ def _report_unsupported_features(features_unavailable):
         uf_db.store_unsupported_features('hare', features_unavailable))
 
 
-def logrotate():
+def logrotate(url: str):
     try:
-        setup_info = get_data_from_provisioner_cli('get_setup_info')
-        if setup_info != 'unknown':
-            server_type = setup_info['server_type']
-            shutil.copyfile(
-                f'/opt/seagate/cortx/hare/conf/logrotate/{server_type}',
-                '/etc/logrotate.d/hare')
+        provider = ConfStoreProvider(url)
+        machine_id = provider.get_machine_id()
+        server_type = provider.get(
+            f'server_node>{machine_id}>type')
+
+        if server_type == 'VM':
```

Comment:
This is error prone, just in case this one string ('VM') changes the code will break. I suggest to have a cortx-py-utils interface that checks the type, e.g. is_setup_virtual() if not already present, @SwapnilGaonkar7 can you please confirm?
May be for now we can land these changes but we must propose to have a generic function. Or you can yourself submit a PR in cortx-py-utils to add such a function.
cc @ujjwalpl, @sachinpunadikar.

Reply:
@mssawant Quick search on 'cortx-utils' suggests that there is no such functions as of now.
@sachinpunadikar Can you please confirm?